app/server/log.ts

```diff
@@ -14,14 +14,3 @@ export const log = winston.createLogger({
     new winston.transports.Console({ format: winston.format.simple() })
   ]
 });
-
-export const audit = winston.createLogger({
```

Comment:
This was never used so I removed it

Reply:
I think there might have been a requirement for an audit logger.

Reply:
@paulbrown1982 - I've merged this as I'd like to have at least some logging working for this app. Let's reintroduce the audit logger in a separate ticket if there is a genuine requirement for this.